### PR TITLE
Improve performance workspace listing

### PR DESCRIPTION
**Pull request by zak39**
OP#3156

### CODE REVIEW

File: src/SpaceDetails.vue
Comment (smarinier) [inline src/SpaceDetails.vue:1]: Are you sure indent is correct on this file ? It looks like wrong from here (tab vs spaces ??)

File: src/SpaceTable.vue
Comment (smarinier) [inline src/SpaceTable.vue:1]: same here

File: src/WorkspaceContent.vue
Comment (smarinier) [inline src/WorkspaceContent.vue:1]: same here

File: src/store/actions.js
Comment (smarinier) [inline src/store/actions.js:1]: same here

**Review by smarinier — COMMENTED** [review]
see comments

**Comment by zak39** [comment]
> see comments

Thanks, I've fixed it :+1:

It's strange though, I will look into where the issue is.

**Pull request merged**